### ISSUE TITLE
Using ${{}} now breaks in 3.9.2

### ISSUE THREAD

**Issue by kensodemann** [ionic-team/ionic-app-scripts#1310]
_From @thierce on November 10, 2017 16:0_

**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

Please do not submit support requests or "How to" questions here. Instead, please use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
If I try to put a dollar sign $ in front of {{somevar}} like this:
${{somevar}}
I get error:
somevar is not defined
or:
Runtime Error
Unexpected token .

Putting a space in between is fine though $ {{somevar}}

This was not a problem prior to 3.9.2

**Expected behavior:**
 ${{somevar}} would produce something like: $3.00

**Steps to reproduce:**
Create a var in a .ts file and use the following in the template:
${{somevar}}

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
@ionic/cli-utils  : 1.18.0
    ionic (Ionic CLI) : 3.18.0

local packages:

    @ionic/app-scripts : 3.1.0
    Ionic Framework    : ionic-angular 3.9.2

System:

    Node : v7.9.0
    npm  : 5.5.1
    OS   : Windows 10
```

_Copied from original issue: ionic-team/ionic#13388_

**Comment by kensodemann** [comment]
_From @nikmartin on November 10, 2017 21:8_

OMG, i have been beating my head on my desk for the past 4 hours because of this! Thank you!
Here's the specific error I get when trying to run ionic build:

```
Error: ./src/pages/prop-info/prop-info.ts
Module parse failed: Unexpected token (44:916)
You may need an appropriate loader to handle this file type.
|     PropInfoPage = __decorate([
|         Component({
|             selector: 'page-prop-info',template:/*ion-inline-start:"/home/nmartin/Projects/GHA/gha-app/src/pages/prop-info/prop-info.html"*/`<ion-header>\n\n  <ion-navbar color="primaryColor">\n    <ion-buttons left>\n      <button ion-button icon-only (click)="viewCtrl.dismiss()">\n        <ion-icon name="arrow-back"></ion-icon>\n      </button>\n    </ion-buttons>\n    <ion-title>Property Info</ion-title>\n  </ion-navbar>\n</ion-header>\n\n<ion-content class="info-page">\n\n  <ion-slides pager="true" paginationType="fraction" zoom="true">\n    <ion-slide *ngFor="let image of property.images">\n      <img width="100%" src="{{image}}" />\n    </ion-slide>\n  </ion-slides>\n\n  <div class="prop-info" padding>\n    <h1>{{property.name}}</h1>\n    <div class="prop-desc">{{property.desc}}</div>\n\n    <ion-list>\n      <ion-item>\n        <ion-icon name="logo-usd" item-start></ion-icon>\n        ${{property.dailyRate}}/day\n      </ion-item>\n      <ion-item>\n        <ion-icon name="paw" item-start></ion-icon>\n        <span *ngFor="let game of property.game; last as isLast">{{getGameName(game, isLast)}}</span>\n      </ion-item>\n      <ion-item>\n        <ion-icon name="map" item-start></ion-icon>\n        {{typeFilter[property.type].name}}\n      </ion-item>\n      <ion-item>\n        <ion-icon name="locate" item-start></ion-icon>\n        <span *ngFor="let weapon of property.weapons; last as isLast">{{getWeaponName(weapon, isLast)}}</span>\n      </ion-item>\n    </ion-list>\n\n    <h3>Rules</h3>\n    <!-- <div *ngIf="!{{property.rules}}">None specified</div> -->\n    <div>{{property.rules}}</div>\n\n  </div>\n\n\n  <ion-fab bottom right>\n    <button ion-fab color="secondaryColor" (click)="reserveProperty(property)">\n      <ion-icon name="arrow-forward"> </ion-icon>\n    </button>\n  </ion-fab>\n</ion-content>\n`/*ion-inline-end:"/home/nmartin/Projects/GHA/gha-app/src/pages/prop-info/prop-info.html"*/
|         }),
|         __metadata("design:paramtypes", [NavController,
 @ ./src/pages/prop-info/prop-info.module.ts 9:0-43
 @ ./src lazy
 @ ./node_modules/ionic-angular/util/ng-module-loader.js
 @ ./node_modules/ionic-angular/util/module-loader.js
 @ ./node_modules/ionic-angular/components/modal/modal-component.js
 @ ./node_modules/ionic-angular/index.js
 @ ./src/app/app.module.ts
 @ ./src/app/main.ts
    at new BuildError (/home/nmartin/Projects/GHA/gha-app/node_modules/@ionic/app-scripts/dist/util/errors.js:16:28)
    at callback (/home/nmartin/Projects/GHA/gha-app/node_modules/@ionic/app-scripts/dist/webpack.js:121:28)
    at emitRecords.err (/home/nmartin/Projects/GHA/gha-app/node_modules/webpack/lib/Compiler.js:269:13)
    at Compiler.emitRecords (/home/nmartin/Projects/GHA/gha-app/node_modules/webpack/lib/Compiler.js:375:38)
    at emitAssets.err (/home/nmartin/Projects/GHA/gha-app/node_modules/webpack/lib/Compiler.js:262:10)
    at applyPluginsAsyncSeries1.err (/home/nmartin/Projects/GHA/gha-app/node_modules/webpack/lib/Compiler.js:368:12)
    at next (/home/nmartin/Projects/GHA/gha-app/node_modules/tapable/lib/Tapable.js:218:11)
    at Compiler.compiler.plugin (/home/nmartin/Projects/GHA/gha-app/node_modules/webpack/lib/performance/SizeLimitsPlugin.js:99:4)
    at Compiler.applyPluginsAsyncSeries1 (/home/nmartin/Projects/GHA/gha-app/node_modules/tapable/lib/Tapable.js:222:13)
    at Compiler.afterEmit (/home/nmartin/Projects/GHA/gha-app/node_modules/webpack/lib/Compiler.js:365:9)
```
weirdly, running ionic build --prod works, but ionic 3.9.2 is also not including js librarys that are included using `import GeoFire from 'geofire';`

**Comment by kensodemann** [comment]
_From @matoos32 on November 12, 2017 5:30_

I had a similar problem, if not the same, after upgrading to Angular 5. Escaping the '$' fixed the issue for me i.e.  `${{somevar}}` --> `\${{somevar}}`

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

I was able to duplicate the `${{somevar}}` issue in an Ionic blank starter.
I also tried it in a new Angular CLI app to see if the issue also occurred there but it did not.

I am moving this to the `@ionic/app-scripts` repo.

Thank you for using Ionic

**Comment by danbucholtz** [comment]
@thierce,

This is fixed in 3.1.1.

Thanks,
Dan
